tests/test_edgeql_fts.py

```diff
@@ -19,24 +19,37 @@
 
 import os.path
 
+import edgedb
+
 from edb.testbase import server as tb
 
 
-class TestEdgeQLFTS(tb.QueryTestCase):
+class TestEdgeQLFTSQuery(tb.QueryTestCase):
```

Comment:
Add a test where the query is produced by some nontrivial computation. Concatenating some strings together might be good enough.

(I *think* it will fail, for reasons I discussed in another comment, but should be easy to fix)

Reply:
Added the test. It does not fail.